packages/shared/src/store/dbHooks.ts

```diff
@@ -379,10 +379,23 @@ export const useChannel = (options: { id: string }) => {
   });
 };
 
-export const usePostWithRelations = (options: { id: string }) => {
+export const usePostWithThreadUnreads = (options: { id: string }) => {
   const tableDeps = useKeyFromQueryDeps(db.getPostWithRelations);
   return useQuery({
     queryKey: [['post', options], tableDeps],
+    staleTime: Infinity,
+    queryFn: () => db.getPostWithRelations(options),
+  });
+};
+
+export const usePostWithRelations = (
```

Comment:
maybe a comment about how these two differ/why we set it up this way if you think we'll stick with this approach